fetch/programmer-humour/script.js

```diff
@@ -0,0 +1,27 @@
+
+let myData;
+
+async function fetchData() {
+    try{
+        const response = await fetch("https://xkcd.now.sh/?comic=latest")
+
+        if (!response.ok) {
+            throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+
+        myData = await response.json()
+        //console.log(`${JSON.stringify(myData)}  this is api`)
```

Comment:
Please, don't leave commented out code in your final work. Other than that look good.